app/registries/foreman/setting_manager.rb

```diff
@@ -95,9 +95,13 @@ def _inline_validates(name, validations)
         validates(name, validations)
       end
 
-      def validates(name, validations)
-        _wrap_validation_if(name, validations)
-        Setting.validates(:value, validations)
+      def validates(name, validations, **opts)
+        if validations.is_a?(Proc)
```

Comment:
Sounds like `validations` would be an array so is this ever true?

Reply:
It can be hash by default, but I want to allow special case